src/device/rdk/applications/launch_with_content.rs

```diff
@@ -132,84 +132,100 @@ pub fn process(_packet: String) -> Result<String, String> {
             app_created = true;
         }
     }
-
-    if app_created {
-        // ****************** Youtube.1.deeplink ********************
-        #[derive(Serialize)]
-        struct Param {
-            url: String,
-        }
-        #[derive(Serialize)]
-        struct RdkRequest {
-            jsonrpc: String,
-            id: i32,
-            method: String,
-            params: Param,
+    let is_cobalt = Dab_Request.appId == "Cobalt" || Dab_Request.appId == "Youtube";
+    let mut param_list = vec![];
+    if (is_cobalt) {
```

Comment:
remove parenthesis to avoid warning "unnecessary parentheses around `if` condition"